### ISSUE TITLE
Storing/updating the config

### ISSUE THREAD

**Issue by nathanmarks** [muibotdev/muibot#3]
@mbrookes 

Where do you want to store the config? In the repo for now? We could always just keep it in the repo as a maintained example of sorts.

How do you want to be able to update, git?

**Comment by mbrookes** [comment]
In the repo makes sense, but see my comment here: https://github.com/muibotdev/muibot/issues/1#issuecomment-191272224

Thoughts?

> How do you want to be able to update, git?

Surely that'll depend on where it's deployed?

**Comment by nathanmarks** [comment]
@mbrookes nope, because it'd be nice to trigger it via a CI completion hook on master (or some other branch)
